tests/ColumnSchemaBuilderTest.php

```diff
@@ -2,12 +2,12 @@
 
 declare(strict_types=1);
 
-namespace yiiunit\framework\db\pgsql;
+namespace Yiisoft\Db\Pgsql\Tests;
 
 use Yiisoft\Db\Schemas\ColumnSchemaBuilder;
 use Yiisoft\Db\Tests\ColumnSchemaBuilderTest as AbstractColumnSchemaBuilderTest;
 
-final class ColumnSchemaBuilderTest extends AbstractColumnSchemaBuilderTest
```

Comment:
Why not `final`?

Reply:
Because I am going to run the tests from each implementation in db, it currently works the other way around, and the idea is to move the tests to each implementation you can read the issue.